packages/ra-ui-materialui/src/input/AutocompleteArrayInput.stories.tsx

```diff
@@ -593,122 +593,3 @@ export const InsideReferenceArrayInputWithCreationSupport = () => (
         <Resource name="books" edit={BookEditWithReferenceAndCreationSupport} />
     </Admin>
 );
-
-const booksDataProvider = fakeRestDataProvider({
```

Comment:
Isn't this code used for tests on the AutocompleteArrayInput?

Also, linter wqrnings.

Reply:
This data was only used for the test I made in [this commit](d02ad81ba29f90c155eae8ce23fd8c5b93819a55), but this test is now in `FisterButton.spec.tsx` and works with the `FisterButton.stories.tsx`